doc/source/examples/analysis/custom_trajectory_analysis.ipynb

```diff
@@ -191,7 +172,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Running the analysis iterates over the trajectory. The output is saved in `rog.results`, which has the same number of rows, as frames in the trajectory."
+    "Running the analysis iterates over the trajectory. The output is saved in `rog.results`, which has the same number of rows, as frames in the trajectory. The output is saved in rog.results.timeseries. Below, a brief example shows that you can access the results both at rog.results.timeseries and rog.results['timeseries']."
```

Comment:
```suggestion
    "Running the analysis iterates over the trajectory. The output is saved in `rog.results.timeseries`, which has the same number of rows, as frames in the trajectory. You can access the results both at `rog.results.timeseries` and `rog.results['timeseries'].`"
```

Could you please add a code cell below this, showing both, please? Currently the cell below only shows `rog.results['timeseries']`.

Reply:
Done. Thanks for the comment!